src/chapters/az-cli/intro_az-cli.rst

```diff
@@ -0,0 +1,102 @@
+
+- feature accessibility
+- workflow efficiency
+- automation capability
+
+The GUI Delay
+=============
+
+A GUI inherently requires additional development work to produce the layouts, buttons, and other components needed for a user to interact with it. Their graphical nature will always be more complex to develop and maintain than a text-based, CLI, counterpart.
+
+This difference seems arbitrary to you as the end user. It may appear to only burden the Azure maintainers. But as the end user you are restricted to only using what has been developed by their teams. Whenever a new resource, service or capability is supported internally by Azure it takes time for both GUI and CLI to have the feature added for use. The process looks something like:
+
+#. internal development of new feature
+#. implement support in the Azure REST API
+#. implement support in the CLI
+#. implement support in the GUI (if at all)
+
+If a hot new Azure feature is released, which will take more time to become available for you to use, the GUI or CLI? Because of the relative complexity GUIs always take more time to develop than making an addition to the CLI. Which means you can expect access to the new feature to be released to the CLI before, if ever, the web interface is updated.
+
+And what about the more granular management tasks that you may need to perform on your resources? It stands to reason that priority in GUI development will always be given to the *most common* needs before spending time developing the graphics to support more niche areas. Again, because of their simplicity, CLIs will have more more comprehensive features added to them that may never even reach the GUI.
+
+.. admonition:: Fun Fact
+
+    The only interface that exposes greater access to Azure resources over the CLI is the REST API itself. After all, a REST API is just another *interface* that happens to use HTTP requests instead of graphics or the command line. You can make requests directly to the REST API through any HTTP client like a browser, ``curl``, or the ``az CLI`` itself. But most of the time it is a tedious process not meant for day-to-day use when compared to using the GUI or CLI.
+
+There are two key takeaways here about a system, of which Azure is but one example, which supports both a GUI and CLI:
+
+- **the CLI will receive the latest additions and updates before the GUI**
+- **the CLI will have more granular management capabilities that aren't present on the GUI**
+
+While the purpose of this lesson is to inspire your understanding and appreciation for CLIs they *are not always the best choice*. Sometimes GUIs offer an abstraction over more tedious work that can make them worth using. Like most things in the development world you should not blindly adhere to a single approach. What is most important is **to select the right tool for the job** that empowers your workflow. 
+
+Work Velocity
```

Comment:
honestly I think combining this section and the previous section into one section would create the length necessary for these topics.